### DISCUSSION TITLE
Is it possible to add interface/flowgate limits?

### DISCUSSION THREAD

**Discussion by aliati94** [PyPSA/PyPSA#1068] [Q&A]
I'm working on building a model that mimics ERCOT, which contains interface limits. An interface is a group of lines, with a specific aggregate flow limit. Is that something currently doable?

**Comment by gincrement** [comment]
Yes, this kind of details can be done via custom constraints. An example is available here: https://docs.pypsa.org/latest/user-guide/optimization/custom-constraints/?h=custom
